src/renderer/renderScene.ts

```diff
@@ -174,6 +174,19 @@ const renderLinearPointHandles = (
   context.strokeStyle = origStrokeStyle;
 };
 
+// From https://github.com/Modernizr/Modernizr/blob/master/feature-detects/emoji.js
+const supportsEmoji = () => {
```

Comment:
Maybe move that to `utils`? https://github.com/excalidraw/excalidraw/blob/master/src/utils.ts

Reply:
Done.